database/pgtsv_to_json

```diff
@@ -3,6 +3,29 @@
 # Usage: pgtsv_to_json COLUMN_NAME1:TYPE1 COLUMN_NAME2:TYPE2 ...
 
 import json, sys, csv, re
+from datetime import datetime
+
+def timestamp(timestamp_str):
+    """Given a timestamp string, return a timestamp string in ISO 8601 format to emulate
+    Postgres 9.5's to_json timestamp formatting. The given timestamp is assumed UTC.
+
+    Time zones are not supported. http://bugs.python.org/issue6641
+
+    Examples:
+
+        >>> timestamp('2016-06-17 20:10:38')
+        '2016-06-17T20:10:38+00:00'
+
+        >>> timestamp('2016-06-17 20:10:37.9293')
+        '2016-06-17T20:10:37.929300+00:00'
+
+    """
+
+    try:
+        parsed = datetime.strptime(timestamp_str, '%Y-%m-%d %H:%M:%S.%f')
+    except ValueError:
+        parsed = datetime.strptime(timestamp_str, '%Y-%m-%d %H:%M:%S')
+    return parsed.isoformat() + '+00:00'
```

Comment:
Timezone-aware dates are stored internally in UTC, but they're output in whichever time zone is set for the session. This can be (a) the system time zone, (b) the time zone set in postgresql.conf, (c) the current time zone setting for the psql session, or (d) [something else](https://www.postgresql.org/docs/8.3/static/datatype-datetime.html#DATATYPE-TIMEZONES). Given that, defaulting to "local" time seems potentially convenient but very easily incorrect (it's not really clear what "local" means in this context).

This code will raise on any timestamp with a timezone, so it's not handling them incorrectly -- it's refusing to handle them at all.

Now that you mention it, it is pretty odd to output a UTC zone offset in this case. It shouldn't output a zone offset at all. I can remove the `+00:00` on output. How does that sound?